data/coalition/coalition culture conversations.txt

```diff
@@ -233,3 +233,137 @@ mission "Coalition Folklore: Kimek"
+		government "Coalition"
+		not attributes "coalition station"
+	destination "Ring of Friendship"
+	on enter
+		conversation
+			`Your ship narrowly avoids a few small Coalition vessels as you exit <origin>'s atmosphere. Not a second later, while you're still stunned by the near-crash, once again the <ship> is almost hit by incoming ships, this time Heliarch warships, in what looks like an intense chase after the ships from before.`
+			`	After they've all passed, you check your comms and radar, finding some other civilian ships nearby seem to have experienced a similar shock of their own. Some of the ships linger over <origin> for a while, as if unsure of what to do. Others bolt away with engines at full power, leaving the system in a blink, and some eventually decide to land again, either too curious about the situation, or perhaps wanting to avoid any trouble.`
+			`	Some more minutes later, you see a select few of the Heliarch ships that passed close to you departing the planet. They pay you no mind, and much to your relief steer a safe distance from your ship this time.`
+
+
+
+mission "Coalition Departing Trouble - B"
+	minor
+	invisible
+	to offer
+		has "Coalition: First Contact: done"
+		random < 2
+	source
+		government "Coalition"
+	destination "Ring of Friendship"
+	on enter
+		conversation
+			`Once you're a good distance from <origin>, your sensors alert you to danger in the system: several explosions and a torrent of projectiles are flying nearby. You swiftly ready yourself and prepare for the threat, only to find Coalition civilian ships blowing up near the middle of the system. Some Heliarch warships are in their midst, moving from one to the other, unbothered by the ship debris as vessels explode around them.`
+			`	They appear to be boarding the few ships that weren't damaged enough to have their hulls ruptured. After a few short minutes, they seem to have seized control of the vessels, and are having them follow the warships out of the system.`
+
+
+
+mission "Coalition Departing Trouble - C"
+	minor
+	invisible
+	to offer
+		has "Coalition: First Contact: done"
+		not "assisting lunarium"
+		not "joined the heliarchs"
+		random < 2
+	source
+		government "Coalition"
+	destination "Ring of Friendship"
+	on enter
+		conversation
+			`You depart to find something of a screen of Heliarch warships around <origin>, circling it and approaching all departing civilian ships. It's no different with your <ship>, and soon enough you're hailed by a Heliarch Punisher approaching your position.`
+			`	"Salutations, Captain <last>. Performing a routine inspection, we are. Board you momentarily we shall, to verify the safety of your vessel, then leave you to your business we shall."`
+			`	They sign off before you can respond, and the Punisher readies to board the <ship>, as more warships gather around the scene. Dozens of armed Heliarch agents march into your ship silently, diligently going about checking the bunks, cockpit, engine room...`
```

Comment:
It might be cool to spawn some NPCs so the player can actually see all the heliarch ships around the planet. Honestly, some of the other ones could use NPCs as well, but I'm not sure if it's worth all the trouble.